### PR TITLE
Revised event model indexes

### PR DESCRIPTION
**Pull request by aminlatifi**

### CODE REVIEW

**Comment by aminlatifi** [comment]
@mohammadranjbarz Please review this pull request. Also, please add a migration file to remove old db indexes

File: src/models/events.model.js [hunk 1]
```diff
@@ -41,9 +42,12 @@ function createModel(app) {
       timestamps: true,
     },
   );
-  event.index({ transactionHash: 1, logIndex: 1, transactionIndex: 1, blockNumber: 1, status: 1 });
```
Comment (mohammadranjbarz) [inline src/models/events.model.js:44]: @aminlatifi 
I think it's good to add an index  for status ` event.index({ status: 1 });

**Review by mohammadranjbarz — CHANGES_REQUESTED** [review]

File: src/models/events.model.js (hunk 1 above)
Comment (aminlatifi) [reply src/models/events.model.js:44]: Index: true was in status field property.

Thanks @mohammadranjbarz.

Comment (mohammadranjbarz) [reply src/models/events.model.js:44]: My mistake, you are right

**Comment by mohammadranjbarz** [comment]
> @mohammadranjbarz Please review this pull request. Also, please add a migration file to remove old db indexes

@aminlatifi 
Ok I'll do it, in this PR

**Comment by mohammadranjbarz** [comment]
@aminlatifi 
I added migration file and after running that it happened to db indexes:

Screen Shot 1400-03-02 at 22 37 31

**Comment by aminlatifi** [comment]
@mohammadranjbarz I add a script to just remove previous indexes. I appreciate your effort to create another script, but there was an issue in running migration

**Comment by mohammadranjbarz** [comment]
> @mohammadranjbarz I add a script to just remove previous indexes. I appreciate your effort to create another script, but there was an issue in running migration

@aminlatifi 
I didn't get your point and the migrations we ok when I ran in my system

**Pull request merged**